protos/shell/shell.proto

```diff
@@ -35,7 +35,8 @@ message ShellResult {
         NO_SYSTEM = 2; // No system is connected
         CONNECTION_ERROR = 3; // Connection error
         NO_RESPONSE = 4; // Response was not received
-        BUSY = 5; // Shell busy (transfer in progress)
+        NOT_COMPLETED = 5; // Response transfer potentially not completed (termination symbol not received)
```

Comment:
Wait, now I'm confused :sweat_smile:. What's the difference between BUSY and NOT_COMPLETED? As a user, do I ever want to differentiate them?

Reply:
Hmm. Yes, this really can confuse.
- `BUSY` - shell is busy now
- `NOT_COMPLETED` - in response of current request was not received `ESC`, so we suppose that transfer was interrupted .

May be need to use more clear name (or may be refuse this enum value)?

Reply:
I think that it is the same thing for a user: the request has not completed, so a new request cannot be sent. I can't think of a case where a user says "if it's BUSY, do this, and if it's NOT_COMPLETED, do that". If there is no use-case for splitting them, I would vote for reusing `BUSY` (if you guys agree, of course).

Reply:
Hmm. I think this is not the same. But now I think that `NOT_FINISHED` will be more likely to mislead the user than to help. 
May be only add some explaining to the `**brief**`? To notice user to try to increase delay time if response looks incomplete

Reply:
ok :slightly_smiling_face: